dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -485,6 +485,11 @@ private void selectMultipleItems(final List<String> rowNameList, final TreeTable
      */
     private void startGetJob(final List<Ds3TreeTableValueCustom> listFiles,
             final Path localPath, final Ds3Client client) {
+        if (listFiles.stream().anyMatch(value -> value.getType() == BaseTreeModel.Type.Bucket)) {
+            LOG.info("Cannot perform GET on bucket");
+            alert.error("noGetBucket", getWindow());
+            return;
```

Comment:
why do we have the same 3 lines of code here again?

Reply:
Because we have multiple methods that do the same thing =(

Reply:
🔢